spec/models/course/announcement_spec.rb

```diff
@@ -4,4 +4,19 @@
   it { is_expected.to belong_to(:creator).class_name(User.name) }
   it { is_expected.to belong_to(:course).inverse_of(:announcements) }
   it { is_expected.to validate_presence_of(:title) }
+
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    context 'when announcement is created' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.